app/cmd/backup.go

```diff
@@ -80,6 +84,11 @@ func BackupCreateCmd() cli.Command {
 				Name:  "storage-class-name",
 				Usage: "Storage class name of the pv binding with the volume",
 			},
+			cli.StringFlag{
+				Name:  "backup-block-size",
+				Usage: "Backup block size quantity",
+				Value: backupstore.DEFAULT_BLOCK_SIZE_QUANTITY,
```

Comment:
Can we rename it to `DEFAULT_BLOCK_SIZE_IN_MB`?

```suggestion
				Value: backupstore.DEFAULT_BLOCK_SIZE_QUANTITY,
```

Reply:
Yes, might be more clear?

Reply:
Refactor the backupstore needed here: longhorn/backupstore/pull/251